### PR TITLE
Gzip integration

### PR DESCRIPTION
**Pull request by DanielChicot**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -70,7 +67,7 @@ services:
     volumes:
       - shared-volume:/opt/snapshot-sender/data
     command: >-
-      --logging.level.root=DEBUG
+      --logging.level.root=INFO
```
Comment (dan-hill2802) [inline docker-compose.yml]: can this come from env var?

File: docker-compose.yml [hunk 2]
```diff
@@ -253,7 +250,7 @@ services:
       S3_STATUS_FOLDER: "status"
       S3_SERVICE_ENDPOINT: "http://s3-dummy:4572"
       NIFI_ROOT_FOLDER: "/data/output" # see the shared-volume above
-      NIFI_FILE_NAMES_CSV: "db.core.addressDeclaration-000001.txt.bz2,db.quartz.claimantEvent-000001.txt.bz2"
+      NIFI_FILE_NAMES_CSV: "db.core.addressDeclaration-045-050-000001.txt.gz,db.quartz.claimantEvent-045-050-000001.txt.gz"
```
Comment (dan-hill2802) [inline docker-compose.yml:251]: where does `045-050` come from?

File: src/integration/kotlin/SnapshotSenderIntegrationTest.kt [hunk 3]
```diff
@@ -73,11 +73,11 @@ class SnapshotSenderIntegrationTest : StringSpec() {
         "Verify for every source collection a finished file was written to s3" {
             logger.info("Checking $bucketUri ...")
 
-            //s3 in:  test/output/db.core.addressDeclaration-000001.txt.bz2.enc
-            //s3 out: test/status/db.core.addressDeclaration-000001.txt.bz2.enc.finished
+            //s3 in:  test/output/db.core.addressDeclaration-000001.txt.gz.enc
```
Comment (dan-hill2802) [inline src/integration/kotlin/SnapshotSenderIntegrationTest.kt:76]: can we just remove the commented out stuff?

File: src/test/kotlin/app/batch/HttpWriterTest.kt [hunk 4]
```diff
@@ -99,7 +99,7 @@ class HttpWriterTest {
         val formattedMessages = logCaptor.allValues.map { it.formattedMessage }
         assertEquals("""Writing items to S3", "number_of_items":"1"""", formattedMessages[0])
         assertEquals("""Checking item to  write", "file_name":"db.core.addressDeclaration-000001.txt.bz2.enc", "full_path":"exporter-output\/job01\/db.core.addressDeclaration-000001.txt.bz2.enc"""", formattedMessages[1])
-        assertEquals("""Found collection of file name", "collection":"db.core.addressDeclaration", "file_name":"exporter-output\/job01\/db.core.addressDeclaration-000001.txt.bz2.enc"""", formattedMessages[2])
+        assertEquals("""Found collection of file name", "collection":"addressDeclaration", "file_name":"exporter-output\/job01\/db.core.addressDeclaration-000001.txt.bz2.enc"""", formattedMessages[2])
```
Comment (dan-hill2802) [inline src/test/kotlin/app/batch/HttpWriterTest.kt:102]: if we remove `db.core` are we specifying the database somewhere else?

**Review by dan-hill2802 — COMMENTED** [review]
Some questions added.

There's also quite a bit of commented out stuff that should probably just go

File: docker-compose.yml (hunk 2 above)
Comment (DanielChicot) [reply docker-compose.yml:251]: It's the range of bytes that the first character of the ids in this batch are in.

File: src/integration/kotlin/SnapshotSenderIntegrationTest.kt (hunk 3 above)
Comment (DanielChicot) [reply src/integration/kotlin/SnapshotSenderIntegrationTest.kt:76]: They are actual comments not commented out code.

File: src/test/kotlin/app/batch/HttpWriterTest.kt (hunk 4 above)
Comment (DanielChicot) [reply src/test/kotlin/app/batch/HttpWriterTest.kt:102]: Yes in the database header.

File: docker-compose.yml (hunk 1 above)
Comment (DanielChicot) [reply docker-compose.yml]: Not really this is a property picked up by spring's logging framework  and this is how it is provided

Comment (DanielChicot) [reply docker-compose.yml]: The main point of this change was to reduce all the noise in the circleci console.

Comment (dan-hill2802) [reply docker-compose.yml]: ah ok, I'd at least create a variable in the docker compose file so that you are only defining the log level the once

Comment (DanielChicot) [reply docker-compose.yml]: Ok done.

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**